client/components/orderable-list/payment-method.js

```diff
@@ -10,36 +10,29 @@ import { Icon, trash } from '@wordpress/icons';
  * Internal dependencies
  */
 import './payment-method.scss';
+import { getPaymentMethodSettingsUrl } from '../../utils';
+
+const PaymentMethod = ( { id, label, description, onDeleteClick } ) => {
+	const settingsUrl = getPaymentMethodSettingsUrl( id );
 
-const PaymentMethod = ( {
-	label,
-	description,
-	onManageClick,
-	onDeleteClick,
-} ) => {
 	return (
 		<>
 			<div className="payment-method__icon" />
 			<div className="payment-method__text">
-				<Button
-					isLink
-					className="payment-method__label"
-					onClick={ onManageClick }
-				>
+				<a href={ settingsUrl } className="payment-method__label">
```

Comment:
It looks like the method name links are `14px` in the designs unless I am missing something! 😄

Reply:
My bad, I didn't check there, I assumed that the previous ones were correct - looks good then! :D

Reply:
Huh, you're right!

Previously the `.components-button` rule took precedence over `.payment-method__label`. This has since "fixed itself" (it was actually fixed by using `<a>` here).